packages/core/src/plugins/index.scss

```diff
@@ -0,0 +1,3 @@
+// ----------------------------------------------------------
```

Comment:
Won't having a single `plugins/index.scss` mean that if we have multiple plugins they will overwrite each other?

Reply:
Yes, the styles will all be imported into this file and may override each other. We are taking care to keep all styles well-scoped within the plugin.